src/Form/listenForFormSubmission.js

```diff
@@ -8,9 +8,9 @@ const listenForGlobalFormSubmission = (forms) => {
             forms.get(e.target)?.submitHandler(),
         true)
 
-        window.addEventListener('keypress', (e) => {
+        window.addEventListener('keydown', (e) => {
```

Comment:
Looks like `keypress` is deprecated. I discovered this because a site was capturing the event before `keypress` would fire. This is better, it fires earlier and is more specific.